network-api/networkapi/wagtailpages/templatetags/homepage_tags.py

```diff
@@ -43,7 +43,7 @@ def get_page_authors(page):
     if page is None:
         return []
     else:
-        all_authors = page.authors.all()
+        all_authors = page.authors.all().prefetch_related("author")
```

Comment:
A N+1 fix to ensure we fetch authors already instead of doing an extra query for each author.